mistletoe/token.py

```diff
@@ -46,18 +46,22 @@ def __repr__(self):
             self.__class__.__name__
         )
 
-        if hasattr(self, "children"):
+        if "children" in vars(self):
             count = len(self.children)
             if count == 1:
                 output += " with 1 child"
             else:
                 output += " with {} children".format(count)
 
-        if hasattr(self, "content"):
+        if "content" in vars(self):
            output += " content=" + _short_repr(self.content)
 
+        if "header" in vars(self):
+           output += " with header"
```

Comment:
It's for all Table tokens, not just the headerless ones. The output can be seen in the test case below. This was added to bring the two mechanisms in sync.

Reply:
OK, let me re-explain: I see header is also output in the AST, yet in there it is _necessary_, because one surely wants to see contents of this table part. In here, in the repr method, seeing `with header`, _when all tables in mistletoe have header_, might be confusing though, or not?

Note that it's not a blocker for me to accept this PR, I'm just not sure whether we do need it. :)

Reply:
Ok, yes I see your point. And since the discussion thread in #60 seems to point towards NOT supporting headerless tables, plus we've already opened up for differences between the two implementations, I'll simply remove this.

Reply:
OK, thanks.